### ISSUE TITLE
Can't Open File

### ISSUE THREAD

**Issue by mschmitinnova** [datto/fireeye-red-team-countermeasure-scanner#1]
I am not sure if I am missing something here, but I am not able to run this. I get a constant stream of errors of not being able to open file.

**Comment by brking8910** [comment]
I   have fond that this is normal when you run the environment variable at 1, it is seeing nothing running memory otherwise you will see output, change your environment variable to 2 by the command $env:usrscanscope = 2 this will run against your home directory (C:) and you should definitely see output there. Change to the 3 or 4 as described in the details of README.

**Comment by Stinteg** [comment]
If you're interested in scanning just the executables in memory, I had to manually set the variable to 1:  $env:usrscanscope = 1

**Comment by Elfystone** [comment]
After a night of digging I was able to find a way to run this without Datto RMM Leaving this here for other MSPs looking for answers. Here is what I did: 

1) Install latest version of Microsoft Visual C++ Redistributable then restart
2) Download Zip file linked from Github by clicking Green "Code" Button > Download Zip
3) Right Click Properties of Zip > Unblock
4) Extract Zip, Open Powershell in admin mode
5) Change the powershell directory to the downloaded folder example: cd C:\Users\(Profile Name)\Downloads\\fireeye-red-team-countermeasure-scanner-main\fireeye-red-team-countermeasure-scanner-main
6) Enter $env:usrscanscope = 2 
(Replace the 2 with 1, 2, 3 or 4 for the following)
 "1" only scans executables running in memory at execution time. "2" scans executables present on the Home Drive (usually C:); "3" scans executables on all fixed and removable drives. "4" scans executables on ALL accessible drives.
7) Run the following: powershell -command "&.\build23.ps1"
8) Powershell will now attempt to run the script, you will see errors scanning certain files, text in red will show exe files it scanned. Takes around 10 mins, a log file should be created in the same directory as where the file was downloaded called log.txt. 

Feel free to reply with any suggestions or improvements. 

It seems this can be ran with command prompt in admin mode if you add powershell -command in front that step 6 code. However it fails to open any of the scanned files
